File: api/src/prescriptions/medicine.go
package prescriptions

// Medicine represents a particular medicine
type Medicine struct {
	Name        string   `json:"name"`
	ID          string   `json:"med_id"`
	Type        string   `json:"type"`
	Appereance  []string `json:"appereance"`
	SideEffects []string `json:"side_effects"`
}

File: iOS/NWHacks/Models/Medicine.swift
//
//  Medicine.swift
//  NWHacks
//
//  Created by Felipe on 2019-01-26.
//  Copyright © 2019 Felipe. All rights reserved.
//

import Foundation

struct Medicine: Codable {
    var Name: String
    var ID: String
    var _Type: String
    var Appereance: [String]
    var SideEffects: [String]
    private enum CodingKeys: String, CodingKey {
        case Name = "name"
        case ID = "med_id"
        case _Type = "type"
        case Appereance = "appereance"
        case SideEffects = "side_effects"
    }
}
